app/mailers/application_mailer.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationMailer < ActionMailer::Base
   default from: "Learn IT Girl <no-reply@learnitgirl.com>"
+  layout 'mailer'
```

Comment:
@pg20 do we need this layout here? cause for now we're sending just plain text in email so no point using layout